### DISCUSSION TITLE
Any update on the newsletter service?

### DISCUSSION THREAD

**Discussion by gloparco** [forwardemail/forwardemail.net#255] [General]
Back in February, you had mentioned that a newsletter service was forthcoming.  It's been a few months now.  Any update on that?  Is it still in the works/being planned?  Any update on when it might be released?  Thank you!

**Comment by titanism** [comment]
It will be released this year, still planned!

**Comment by ceonelson** [comment]
@titanism do you have any ETA for the release? I need to get a newsletter service setup in the next month or so and would love to go through you. 

If it will be later, what would you recommend to use in the meantime that we could import from when you launch?

**Comment by shaunwarman** [comment]
@ceonelson @gloparco 

There is now newsletter and bounce webhook support! We recommend [listmonk](https://listmonk.app/) for now to manage newsletters / campaigns. You can find more information here: https://discuss.privacyguides.net/t/forward-email-email-provider/13370/209

We'll get this added into the docs.

**Comment by titanism** [comment]
Just an update, we're waiting on a PR in ListMonk to get added out of the box.

See <https://github.com/knadh/listmonk/pull/2016>
